src/styles/components/_home-section.scss

```diff
@@ -61,18 +60,15 @@
   }
 
   &--light {
-    background-color: blueviolet;
-
-    @include respond-above('md') {
-      height: rem(500px);
-    }
+    height: auto;
+    background-color: transparent;
 
-    @include respond-above('lg') {
-      height: rem(650px);
-    }
+    & > *:not(:last-child) {
+      margin-bottom: rem(16px);
 
-    @include respond-above('xl') {
-      height: rem(800px);
+      @include respond-above('md') {
+        margin-bottom: rem(30px);
+      }
```

Comment:
Also try this since they are composable now. No need for `popular` modifier.
```
.home-section {
  height: auto;

  &--dark {// dark color}

  &--light {// light color}
}
```